paasta_tools/tron_tools.py

```diff
@@ -559,6 +559,19 @@ def get_node(self):
     def get_schedule(self):
         return self.config_dict.get("schedule")
 
+    def get_cron_expression(self) -> Optional[str]:
+        schedule = self.config_dict.get("schedule")
+        if (
+            isinstance(schedule, Dict)
```

Comment:
does this work with `isinstance(schedule, dict)`? imo, this would be more familiar to folks if that does work